bin/xud

```diff
@@ -53,6 +53,18 @@ const { argv } = require('yargs')
       describe: 'Data directory for lnd',
       type: 'string',
     },
+    'lnd.certificate': {
+      describe: 'Certificate for lnd',
+      type: 'string',
+    },
+    'lnd.host': {
```

Comment:
I think we'd prefer to separate port and host by adding a `port` config option. Adding host/port for the lnd client is part of #58, actually, but lets keep that as part of this PR since it's more focused.